cfile-to-asm-builder/SConstruct

```diff
@@ -1,3 +1,7 @@
+# Skip initializing any tools in the DefaultEnvironment 
+# as we're not using it.
+DefaultEnvironment(tools=[])

```

Comment:
Really don't think these were necessary for simple examples.

Reply:
They're not except to address the issue brought up in the mailing list..

Reply:
And I guess likely not a bad thing for any Windows based builds..